.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,62 @@
+name: Bug Report
+description: Create a report to help us reproduce and correct the bug
+labels: ['bug', 'needs-triage']
+
+body:
+- type: markdown
+  attributes:
+    value: >
+      #### Before submitting a bug, please make sure the issue hasn't been already
+      addressed by searching through [the past issues](https://github.com/probabl-ai/skore/issues).
+- type: textarea
+  attributes:
+    label: Describe the bug
+    description: >
+      A clear and concise description of the bug.
+  validations:
+    required: true
+- type: textarea
+  attributes:
+    label: Steps/Code to Reproduce
+    description: |
+      Please add a minimal code example that can reproduce the error when running it. Be as succinct as possible, **do not depend on external data files**.
```

Comment:
As users may have to give some sort of dataset if they encounter bugs while creating items, shouldn't we encourage them to use stackblitz or build a repo if they need to add "attachements" ?